apps/web/src/services/dashboards/widgets/_components/WidgetDataTable.vue

```diff
@@ -66,7 +68,7 @@
                                 @click="handleClickRow({rowIndex, item})"
                             >
                                 <td v-for="(field, colIndex) in props.fields"
-                                    :key="`td-${props.widgetKey}-${rowIndex}-${colIndex}`"
+                                    :key="`td-${props.widgetKey}-${rowIndex}-${colIndex}-${tableWidth}`"
```

Comment:
praise: wow

Reply:
Question: I can't fully understand. Why did you use `tableWidth` in key?

Reply:
> Question: I can't fully understand. Why did you use tableWidth in key?

To rerender that element when `tableWidth` is updated so the tooltip can be shown when it is ellipsised.

Reply:
I think she was expecting to re-render when the key changed..?!